Mathlib/Combinatorics/SimpleGraph/Connectivity/WalkCounting.lean

```diff
@@ -209,11 +209,10 @@ lemma ConnectedComponent.odd_card_supp_iff_odd_subcomponents {G'} [DecidableRel
 
 lemma odd_card_iff_odd_components : Odd (Nat.card V) ↔
     Odd (Nat.card ({(c : ConnectedComponent G) | Odd (Nat.card c.supp)})) := by
-  cases isEmpty_or_nonempty V with
-  | inl h => simp only [Nat.card_of_isEmpty, Nat.card_eq_fintype_card, Fintype.card_eq_zero,
+  cases' isEmpty_or_nonempty V with _ h
```

Comment:
I don't think this is relevant anymore since I reverted back on the old proof to get rid of the dependency on `[DecidableEq V]`, but I'll keep it in mind for future applications